pyproject.toml

```
@@ -80,6 +80,7 @@ sqlalchemy = { version = "^2.0.18", optional = true }
 getdaft = { version = ">=0.2.12", optional = true }
 cachetools = "^5.5.0"
 pyiceberg-core = { version = "^0.4.0", optional = true }
+polars = "^1.21.0"
```

this should be optional
```suggestion
polars = { version = "^1.21.0", optional = true }
```